labs/lab-2-javascript-game/script.js

```diff
@@ -0,0 +1,124 @@
+var playerHP = 40;
+var grantHP = 10;
+var playersName = "";
+var grantDeaths = 0;
+
+function startGame() {
+	 wouldYou = prompt("Would you like to play?");
```

Comment:
Watch your indentation, here. That's a nine-space indent.

(I'd also recommend either 2 or 4 space indentation, rather than 8, which is what you're using in the rest of this file. 8 is not unheard-of, but it's a little eccentric. You can adjust this in your editor's settings.)